src/baklava/CustomNode.vue

```diff
@@ -123,6 +130,46 @@ export default class CustomNode extends Components.Node {
     items: this.getContextualMenuItems(),
   };
 
+  private currentErrors: IrError[] = []
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
I think this is not necessary now because you do use `this` in the method

Reply:
yes I forgot to remove it